python/dask_cudf/dask_cudf/backends.py

```diff
@@ -447,10 +447,26 @@ def _default_backend(func, *args, **kwargs):
 
 
 try:
+    from dask.dataframe.backends import DataFrameBackendEntrypoint, PandasBackendEntrypoint
```

Comment:
Can we add a TODO here to drop this try/catch during dask version pinning

Reply:
I suppose I can remove this try/except right now? `DataFrameBackendEntrypoint` and `PandasBackendEntrypoint` have been in Dask since October.

Reply:
Yup then good to remove 👍